indico/modules/events/registration/client/js/components/participant-list/ParticipantList.tsx

```diff
@@ -0,0 +1,131 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import participantListPreviewURL from 'indico-url:event_registration.manage_participant_list_preview';
+import manageRegFormListURL from 'indico-url:event_registration.manage_regform_list';
+
+import React from 'react';
+import {Button, Divider, Header, HeaderContent, Icon, Popup} from 'semantic-ui-react';
+import HeaderSubHeader from 'semantic-ui-react/dist/commonjs/elements/Header/HeaderSubheader';
```

Comment:
Isn't this available as `Header.SubHeader`?

Reply:
`<Header sub>` too

Reply:
Fair, although not relevant anymore because I removed the new title